modules/machine-config-daemon-metrics.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-machine-config-daemon-metrics.adoc
+
+[id="machine-config-daemon-metrics_{context}"]
+= Machine Config Daemon metrics
+
+Beginning {product-title} 4.3, the Machine Config Daemon provides a set of metrics. These metrics can be accessed using the Prometheus Cluster Monitoring stack.
+
+The following table describes this set of metrics. Note that:
+
+* Metrics marked with `*` in the `Name` and `Description` columns represent serious errors that might cause performance problems. Such problems might prevent updates and upgrades from proceeding.
+* While some entries contain commands for getting specific logs, the most comprehensive set of logs is available using the `oc adm must-gather` command.
+
+[cols="1,1,2,2", options="header"]
+.MCO metrics
+|===
+|Name
+|Format
+|Description
+|Notes
+
+|mcd_host_os_and_version
+|[]string{"os", "version"}
+|Shows the OS that MCD is running on, such as RHCOS or RHEL. In case of RHCOS, the version is provided.
+|
+
+|ssh_accessed
+|counter
+|Shows the number of successful SSH authentications into the node.
+|The non-zero value shows that someone might have made manual changes to the node. Such changes might cause irreconcilable errors due to the differences between the state on the disk and the state defined in the machine configuration.
+
+|mcd_drain*
+|{"drain_time", "err"}
+|Logs errors received during failed drain. *
+|While drains might need multiple tries to succeed, terminal failed drains prevent updates from proceeding. The `drain_time` metric, which shows how much time the drain took, might help with troubleshooting. For further investigation, see the logs by running:
+
+`$ oc logs -f -n openshift-machine-config-operator machine-config-daemon-<hash> -c machine-config-daemon`
+
+|mcd_pivot_err*
+|[]string{"pivot_target", "err"}
+|Logs errors encountered during pivot. *
+|Pivot errors might prevent OS upgrades from proceeding. For further investigation, run these commands to access the node and see its logs:
+
+`$ oc debug node <node>`
```

Comment:
This is out of order.  If we do `oc debug node` and then try to do `oc logs` I don't think that will work.

The explicit commands would look like:

```
$ debug node/ip-10-0-139-75.us-west-2.compute.internal
$ chroot /host
$ journalctl -u pivot.service
```

This connects to the node, `chroot` to the node's filesystem, then run's the `journalctl` command on the node.

Reply:
Changed accordingly. So how to describe the difference between logs from `oc logs` and from `journalctl`? I want to be more explicit than just "You can also run:".

Reply:
The `oc logs` command is only going to fetch the logs from the particular container in the pod (`machine-config-daemon`).  It's something that you want to run from your workstation, not from a node on the cluster.

The `oc debug node` route gets you on the host itself and using `journalctl` is going to give you the full contents of the journal on the node.  You could find the `machine-config-daemon` logs in that journal, but they will be mixed in with the rest of the node logs and therefore harder to sift through.

Reply:
@rh-max @miabbott one way to make this clear would be to use a different shell prompt for the on-node commands so oc debug node stays the same but instead of using `$` for the chroot and journalctl commands you can use something else like `%` or `#` to show that it's a different level than `$`

Reply:
so it would look like:
```
$ oc debug node/ip-10-0-139-75.us-west-2.compute.internal
% chroot /host
% journalctl -u pivot.service
```
You can also run
`$ oc logs -f ....`

Reply:
@kikisdeliveryservice has the right idea....using different prompts to indicate on your workstation vs on the node itself.  thanks Kirsten!

Reply:
@rh-max wdyt? ^^^

Reply:
Thanks @miabbott and @kikisdeliveryservice ! I've applied your feedback and improved the structure a bit. Does the result look good to you?
(We can't just opt into using the `%` for shell prompt in the docs, so I've separated the "before login" and "after login" steps with words.)

Reply:
The user can also run `oc debug node/<node> -- chroot /host journalctl -u pivot.service`.  With this one liner we won't need to distinguish between the different prompts.

Reply:
Thanks @mike-nguyen , I changed the commands to your simplified command. Since the feedback is implemented and I hear no NOACKs, I'm asking @sheriff-rh to merge this.